src/Elcodi/Component/EntityTranslator/Form/Type/TranslatableFieldType.php

```diff
@@ -189,9 +191,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
      */
     public function evaluateRequired($required, $locale)
     {
-        return $this->fallback
-            ? $this->masterLocale === $locale
-            : $required;
+        return (boolean) $required
```

Comment:
This tertiary looks quite hard to read, don't know if we can improve this one with `if`.